extra-mono/msbuild/autobuild/build

```diff
@@ -1,16 +1,8 @@
-bsdtar xvf msbuild_${PKGVER}+${EXTRAVER}_all.deb
-bsdtar xvf data.tar.xz
-chmod -R g-w usr
-cp -arv usr "$PKGDIR"/
+abinfo "Build MSBuild"
+make
 
+abinfo "Deploy files"
 mkdir -p "$PKGDIR"/usr/bin/
-mkdir -p "$PKGDIR"/usr/share/doc/msbuild/
-
-mv -v "$PKGDIR"/bin/* "$PKGDIR"/usr/bin/
-mv -v "$PKGDIR"/lib "$PKGDIR"/usr/
-mv -v "$PKGDIR"/share/doc/msbuild/copyright "$PKGDIR"/usr/share/doc/msbuild/
-rm -rf "$PKGDIR"/{share,bin}
-
-ln -sv ../../../../dotnet/sdk/2.0.0/Roslyn \
-    "$PKGDIR"/usr/lib/mono/msbuild/15.0/bin
-
+./install-mono-prefix.sh "$PKGDIR"/usr/bin/
+mkdir -p "$PKGDIR"/usr/share/doc/"$PKGNAME"
+install -Dm644 "$SRCDIR"/LICENCE "$PKGDIR"/usr/share/doc/"$PKGNAME"
```

Comment:
You dont need the last two lines - Autobuild3 will do this automatically if it finds "$SRCDIR"/COPYING or "$SRCDIR"/LICENCE.

Reply:
I love that.